llarp/router/router.cpp

```diff
@@ -139,30 +139,45 @@ namespace llarp
     // Compute all stats on all path builders on the default endpoint
     // Merge snodeSessions, remoteSessions and default into a single array
     std::vector<nlohmann::json> builders;
-    auto snode_sessions = services["default"]["snodeSessions"];
+
+    auto serviceDefault = services["default"];
+    builders.push_back(serviceDefault);
+
+    auto snode_sessions = serviceDefault["snodeSessions"];
```

Comment:
Most of these should probably be `variable.at("key")` rather than `variable["key"]`; the latter will create an element (with a default null value) if the key doesn't already exist, while the former checks and throws in such a case (and I believe most of these will always be present, so throwing if they aren't seems acceptable).